### ISSUE TITLE
Wrong highlight of the source of error.

### ISSUE THREAD

**Issue by DeLaGuardo** [tonsky/Clojure-Sublimed#79]
It looks like a corner case, btw. If there is an error after evaling the `defprotocol` form with a problem, e.g. var mapped to the java interface, Sublime highlights only the comment placed above protocol declaration and not the source of the Exception.

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/542042/199965758-aa303511-7faf-491b-be2c-67c1c17b7e8a.png">

Code to reproduce:

```clojure
;; Readable is already mapped to java interface java.lang.Readable
(defprotocol Readable
  (read [this search-clause]))
```

**Comment by tonsky** [comment]
Might be Clojure problem (reporting wrong line number) or nREPL or mine :) Will look into it, thanks

**Comment by tonsky** [comment]
Should be fixed in 6daeabc68af9d6b7492d2d59a6a63241a641b7f5

<img width="1182" alt="Screenshot 2023-01-04 at 20 50 04" src="https://user-images.githubusercontent.com/285292/210637417-96202eaa-6c89-4606-9d30-299b6035d5bd.png">
